superset-frontend/src/dataMask/reducer.ts

```diff
@@ -128,6 +131,7 @@ const dataMaskReducer = produce(
             [],
           cleanState,
           draft,
+          undefined,
```

Comment:
Why do we pass `undefined` here? `fillNativeFilters` seems to accept only 4 arguments

Reply:
This was a result of a previous refactor where I had an additional parameter but ended up reverting. Will remove it. Nice catch!